packages/workspace/src/generators/new/generate-preset.ts

```diff
@@ -93,20 +97,29 @@ function getPresetDependencies(preset: string, version?: string) {
 
     case Preset.AngularMonorepo:
     case Preset.AngularStandalone:
-      return { dependencies: { '@nrwl/angular': nxVersion }, dev: {} };
+      return {
+        dependencies: { '@nrwl/angular': nxVersion },
+        dev: {
+          '@angular-devkit/core': angularCliVersion,
+          '@angular-devkit/schematics': angularCliVersion,
+          '@schematics/angular': angularCliVersion,
+          typescript: typescriptVersion,
+        },
+      };
 
     case Preset.Express:
       return { dependencies: {}, dev: { '@nrwl/express': nxVersion } };
 
     case Preset.Nest:
-      return { dependencies: {}, dev: { '@nrwl/nest': nxVersion } };
+      return {
+        dependencies: {},
+        dev: { '@nrwl/nest': nxVersion, typescript: typescriptVersion },
+      };
 
     case Preset.NextJs:
       return { dependencies: { '@nrwl/next': nxVersion }, dev: {} };
 
     case Preset.ReactMonorepo:
-      return { dependencies: {}, dev: { '@nrwl/react': nxVersion } };
-
     case Preset.ReactStandalone:
       return { dependencies: {}, dev: { '@nrwl/react': nxVersion } };
```

Comment:
Doesn't this need typescript?

Is it because we have a `js` option?

1. I'm not sure we have the ability for them to pass the `--js` option
2. If there is the ability.. we could maybe check for the option here?

Reply:
Yeah, it's because of the `js` option.

I'm hesitant of doing so, I think it's very fragile to assume:

- Whether the option is passed
- The default value in the preset generator if not passed

Any of those things are controlled somewhere else and if they change, it's hard to know that the logic here would need to be changed. If we still decide to do such assumptions, we could do the same and assume whether `@nrwl/jest`, `nrwl/cypress`, `@nrwl/js` need to be installed.